common.js

```diff
@@ -2,15 +2,12 @@ export const supportedDomains = [
     'https://portal.azure.com',
     'https://ms.portal.azure.com',
     'https://rc.portal.azure.com',
-    'https://preview.portal.azure.com',
```

Comment:
I think we actually use this. cc: **@timothywarner-org/pygoat-devs** where you at?